packages/odyssey/src/scss/components/_select.scss

```diff
@@ -0,0 +1,13 @@
+.ods-select:not([data-type]) {
+  @include input-baseline;
+
+  background-image: get-icon('caret', $text-body);
+  background-repeat: no-repeat;
+  background-position: right $em-small-spacing center;
+  background-size: ms(-1);
+  appearance: none;
+
+  option {
+    padding: 0;
```

Comment:
Why?

Reply:
Some browsers add "secret" padding to their dropdown list that also shows up in the `select` element itself. This doesn't remove it entirely because there's still some inaccessible styling that goes on behind the scenes, but it helps.

I'd be fine removing this if it causes any issues.